.github/TEMPLATE_SYNC_PAT.md

```
@@ -0,0 +1,56 @@
+              github_token: ${{ secrets.GITHUB_TOKEN }}
+              source_repo_path: <owner/repo>
+              upstream_branch: <target_branch> # defaults to main
+              pr_labels: <label1>,<label2>[,...] # optional, no default
+    ```
+
+4. If you encounter the error `pull request create failed: Actions is not permitted to create or approve pull requests (createPullRequest)`, follow these additional steps:
+
+    - Go to your project’s **Settings** > **Actions** > **General**.
+    - Under the **Workflow permissions** section, check the box for **Allow GitHub Actions to create and approve pull requests**.
+
+Following these steps should resolve any permission issues with workflows, allowing smooth synchronization between repositories.
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)